pkg/provider-local/node/certs.d/europe-docker.pkg.dev/hosts.toml

```diff
@@ -0,0 +1,6 @@
+# europe-docker.pkg.dev upstream is required for loading the hyperkube image.
+# Further registries are supposed to be added via provider-local
+server = "http://europe.docker.pkg.dev"
```

Comment:
Nope, indeed we don't, see https://github.com/gardener/gardener/pull/11010. 

> It would be nice to still have the webook path being tested, i.e. not remove it from the ensurer.

We won't have anything to inject into this webhook anymore. This `europe.docker.pkg.dev` will be moved to the `reconcile` OSC only, and the `localhost` will become part of the `node` image.